docs/getting_started.md

```diff
@@ -86,7 +100,70 @@ $ ./prometheus --config.file=./documentation/examples/prometheus.yml
 And Grafana at the second:
 
 ```sh
-$ docker run -d --network=cortex -p 3000:3000 grafana/grafana
+$ docker run -d --name=grafana --network=cortex -p 3000:3000 grafana/grafana
+```
+
+In [the Grafana UI](http://localhost:3000) (username/password admin/admin), add a Prometheus datasource for Cortex (`http://cortex2:9009/api/prom`).
+
+## High availability with repication
+
+In this last demo we'll show how Cortex can replicate data among three nodes,
+and demonstrate Cortex can tolerate a node failure without affecting reads and writes.
+
+First clean up the last demo and run a new Consul and Grafana:
```

Comment:
Perhaps add a note here saying you will still need a `cortex` network, and a Prometheus remote_write config pointed to localhost:9001? Skipping straight to this step does not work well right now.